sentences/pt-br/light_HassLightSet.yaml

```diff
@@ -2,4 +2,52 @@ language: pt-br
 intents:
   HassLightSet:
     data:
-      - sentences: []
+      # brightness
+      - sentences:
+          - "<mudar_valor_numerico> [a] luminosidade <no_nome> (para|em) <luminosidade>"
+          - "<mudar_valor_numerico> em <luminosidade> [a] luminosidade <no_nome>"
+          - "<mudar_valor_numerico> <nome> para <luminosidade>"
+        requires_context:
+          domain: "light"
+        response: brightness
+
+      - sentences:
+          - "<mudar_valor_numerico> [a] luminosidade <na_area> para <luminosidade>"
+          - "<mudar_valor_numerico> em <luminosidade> [a] luminosidade <na_area>"
+          - "<mudar_valor_numerico> <area> para <luminosidade>"
+          - "<mudar_valor_numerico> <area> para <luminosidade> de luminosidade"
+        slots:
+          name: "all"
+        response: brightness
+
+      # Max/Min brightness
+      - sentences:
+          - "<mudar_valor_numerico> [a] luminosidade <no_nome> [no] {brightness_level:brightness}"
+          - "<mudar_valor_numerico> [no|em] {brightness_level:brightness} [a] luminosidade <no_nome>"
+        requires_context:
+          domain: "light"
+        response: brightness
+
+      - sentences:
+          - "<mudar_valor_numerico> [a] luminosidade <na_area> [no] {brightness_level:brightness}"
```

Comment:
Suggestion: replacing `[no]` with `[no|em|para]` since it makes sense to say:

`Ajustar luminosidade na sala para 10` or even 
`Ajustar luminosidade na sala em 10`